client/extensions/woocommerce/state/sites/review-replies/reducer.js

```diff
@@ -23,3 +32,43 @@ export function repliesUpdated( state, action ) {
 		[ reviewId ]: replies,
 	};
 }
+
+export function replyDeleted( state, action ) {
+	const { reviewId, replyId } = action;
+	const existingReplies = state || {};
+
+	if ( ! reviewId || ! replyId ) {
+		return { ...existingReplies };
+	}
+
+	const repliesForReview = existingReplies[ reviewId ];
+	const updatedReplies = reject( repliesForReview, { id: replyId } );
+
+	return {
+		...existingReplies,
+		[ reviewId ]: updatedReplies,
+	};
+}
+
+export function replyUpdated( state, action ) {
+	const { reviewId, replyId, reply } = action;
+	const existingReplies = state || {};
+
+	if ( ! reviewId || ! replyId ) {
+		return { ...existingReplies };
```

Comment:
Same q as above

Reply:
Also, do you want to protect against this being called without an action.reply ?